src/main/java/cn/nukkit/Server.java

```diff
@@ -2104,6 +2104,7 @@ private void registerBlockEntities() {
         BlockEntity.registerBlockEntity(BlockEntity.JUKEBOX, BlockEntityJukebox.class);
         BlockEntity.registerBlockEntity(BlockEntity.SHULKER_BOX, BlockEntityShulkerBox.class);
         BlockEntity.registerBlockEntity(BlockEntity.BANNER, BlockEntityBanner.class);
+        BlockEntity.registerBlockEntity(BlockEntity.MUSIC, BlockEntityNoteBlock.class);
```

Comment:
Why its called MUSIC?

Reply:
> Why its called MUSIC?

The block entity that can be called MUSIC is only it. Very strange, but mojang...

Reply:
Yes, this is the correct terminology.